ansi/parser/transition_table.go

```diff
@@ -116,7 +117,8 @@ func GenerateTransitionTable() TransitionTable {
 	table.AddRange(0x00, 0x17, GroundState, ExecuteAction, GroundState)
 	table.AddOne(0x19, GroundState, ExecuteAction, GroundState)
 	table.AddRange(0x1C, 0x1F, GroundState, ExecuteAction, GroundState)
-	table.AddRange(0x20, 0x7F, GroundState, PrintAction, GroundState)
+	table.AddRange(0x20, 0x7E, GroundState, PrintAction, GroundState)
+	table.AddOne(0x7F, GroundState, ExecuteAction, GroundState)
```

Comment:
I would add constant for each key


```suggestion
	table.AddOne(keyDelete, GroundState, ExecuteAction, GroundState)
```

```go
const keyDelete = 0x7F
```

It would avoid these magical numbers

Reply:
Wanted to import `ansi` and use the consts from there, but that would introduce an import cycle, this is fine since all of these are single bytes i.e. have a range from 0x00-0xFF.